File: Data Structures & Algorithms/duplicate-integer/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        let seen = new Set();
      for(let num of nums){
        if(seen.has(num)){
            return true;
        }
        seen.add(num);
          
        }
          return false;
      }
    }

let obj = new Solution();
console.log(obj.hasDuplicate([1, 2, 3, 4]));     
console.log(obj.hasDuplicate([1, 2, 3, 4, 4])); 

File: Data Structures & Algorithms/is-subsequence/submission-0.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
        a = 0
        b = 0
        while len(s) > a and len(t) > b:
            if s[a] == t[b]:
                a += 1

            b += 1

        return len(s) == a